src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -529,48 +556,85 @@ protected override void StopProcessing()
         /// </summary>
         private MemoryStream _ms;
 
+        private StringWriter _sw;
+
+        private Serializer _cli_serializer;
```

Comment:
I don't think there's anywhere currently in the codebase we use underscores in the middle of a symbol name, except perhaps with some of the older styles of naming constants. 
```suggestion
        private Serializer _cliSerializer;
```